src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,17 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [],
+};
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    addBook: (state, action) => {
+      state.books.push(action.payload);
+    },
+  },
+});
```

Comment:
- [ ] Let's be reminded that in _bookSlice_, we are required to create two reducers, namely  `addBook` and `removeBook`. Let's, therefore, ensure to add the `removeBook` reducer.